code/modules/mob/living/carbon/human/human.dm

```diff
@@ -947,3 +947,15 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 			riding_datum.unequip_buckle_inhands(M)
 			riding_datum.restore_position(M)
 	. = ..(M, force)
+
+/mob/living/carbon/human/update_rest_icon(mob/living/M)
+	var/obj/screen/lay/byond_sucks

```

Comment:
Can we rename this something more meaningful?

Reply:
Byond sucks